pennylane_sf/fock.py

```diff
@@ -124,8 +122,8 @@ class StrawberryFieldsFock(StrawberryFieldsSimulator):
 
     _circuits = {}
 
-    def __init__(self, wires, *, analytic=True, cutoff_dim, shots=1000, hbar=2):
-        super().__init__(wires, analytic=analytic, shots=shots, hbar=hbar)
+    def __init__(self, wires, *, shots=None, cutoff_dim, hbar=2):
```

Comment:
:thinking: Is it allowed to have a non-default argument (`cutoff_dim`) after a default one?

Reply:
(from some local testing): ohhh, so the `*` is key? In doing so, we are saying that `wires` is the only positional argument, and while `shots` and `hbar` have defaults, `cutoff_dim` is a non-default keyword argument?

Reply:
Yep the `*` is key!

I don't remember the reason why the SF plugin is like this 😆